README.md

```diff
@@ -213,3 +214,144 @@ If you'd like to do this process manually, then:
    * Open terminal in directory `doc/doxygen`
    * `sh RunDoxygen.sh`
    * In newly created `html` folder, open index.html
+
+
+# 7. Adding New Examples<a name="add-examples" />
+
+As described [above](#examples), you can easily run OpenUxAS examples using the `run-example` script.
+You can also easily add new examples to OpenUxAS that will work with the `run-example` script.
+
+There are three essential parts of an example:
+
+1. the directory that contains the example;
+2. the input files that provide example data; and
+3. the YAML file that describes the details of the example.
```

Comment:
For number 3, perhaps add an adjective so that the same level of description is achieved as for numbers 1 and 2.

Reply:
How about "the YAML file that describes the details of how the example should be configured and run"?

Reply:
That would be good indeed.